packages/extensions-sdk/src/cli/commands/validators/check-directus-config.ts

```diff
@@ -0,0 +1,115 @@
+import { EXTENSION_PKG_KEY, EXTENSION_TYPES, API_EXTENSION_TYPES } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+import { isObject } from '@directus/utils';
+
+const checkDirectusConfig = {
+	name: 'directus-config',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Checking package file exists';
+
+		if (!(await fse.pathExists(`${process.cwd()}/package.json`))) {
```

Comment:
Similar comment as above, with `path.resolve`. Let's also save it into a `const` because we need it further down.